dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -272,6 +272,8 @@ class BGPPeerMgr(Manager):
             [
                 ("meta", "localhost/bgp_asn"),
                 ("neigmeta", ""),
+                ("local_addresses", ""),
```

Comment:
do you still need it here?

Reply:
Yes, it's necessary. 
Because These items are for triggering the failed message to be resent and the message order is not guaranteed.
For example, if the peer setting message come first, and we try to get the interface but that interface has not been set, so this message will fail and it will be cached in this [message queue](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-fpm-frr/bgpcfgd#L198). After the interface was set, this failed message will never be retried.